core/Plugin/Manager.php

```diff
@@ -1076,11 +1076,20 @@ private function installPluginIfNecessary(Plugin $plugin)
 
         if ($saveConfig) {
             PiwikConfig::getInstance()->forceSave();
+            $this->clearCache($pluginName);
```

Comment:
This was actually a bug, when a plugin was installed we sometimes did not clear the caches. Eg when installing core plugins